string[6] joint_names
float32[6] goal_position
float32[6] goal_velocity
float32[6] goal_effort
int32[6] control_mode
int32[6] interrupt
